apps/web/src/components/user/dashboard/index.tsx

```diff
@@ -82,9 +82,10 @@ export default async function Dashboard({ user }: Props) {
               Joined {getRelativeTime(user.createdAt)}
             </p>
             <div className="mb-4 mt-2 w-full text-center text-sm md:w-64 md:text-start">
-              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Minima temporibus maiores
-              nesciunt hic dolores aspernatur corrupti laboriosam consectetur. Vitae, consectetur?
-              <ReactMarkdown>{user.bio}</ReactMarkdown>
+              { !user.bio?
```

Comment:
Could we just have it as `{user.bio}` and it will just remain empty if it's `null`? I checked GitHub and it looks like that's what they do 👍🏻

So no ternary but I'd just double check how it looks when `null` is passed as a child to the `ReactMarkdown` component :shipit: